pyrsistent/_pmap.py

```diff
@@ -140,7 +140,9 @@ def __str__(self):
 
     def __hash__(self):
         # This hashing algorithm is probably not the speediest
```

Comment:
There are assorted comments around pyrsistent about "We should probably cache the hash".  I think this comment _might_ be about adding the cache.

If it _is_ actually about choosing to hash a frozenset of iteritems rather than something else (a tuple of iteritems?, XORing the result of hashing each key and value directly here) then I think this comment should be moved to within the if block.